src/commands/oas2ts.js

```diff
@@ -22,10 +22,11 @@ export const runCommand = async (
 ) => {
   try {
     const silentLogger = pino({ level: 'silent' })
-    runOas2JsonCommand(openApiPath, TEMP_FOLDER, silentLogger)
+    const schemasDir = `${TEMP_FOLDER}/components.schemas`
```

Comment:
ideally use path utilities to join paths, so to avoid using platform specific path separators